pkg/queue/manager.go

```diff
@@ -272,13 +272,14 @@ func (m *Manager) addOrUpdateWorkload(w *kueue.Workload) bool {
 	if q == nil {
 		return false
 	}
-	q.AddOrUpdate(w)
+	wInfo := workload.NewInfo(w)
+	q.AddOrUpdate(wInfo)
```

Comment:
nit: single line might be enough.

Reply:
No. we also need to push `wInfo` into `clusterQueue`